test/emitter.test.ts

```diff
@@ -0,0 +1,88 @@
+// Note: we need to import logger from the root
+// because this is the logger used in logError in ../src/common/util
+import { logger } from "../node_modules/@coder/logger"
```

Comment:
If we `import { logger } from "@coder/logger"` it should pull the same module right?

Reply:
hmm I don't believe so. based on my understanding, I think because we have two separate package.json's, it will look for `@coder/logger` in `/test` which is not the one used inside `/src/common/emitter` so we have to import from above to ensure we're mocking the one used by functions located in `/src`

Reply:
There is no `@coder/logger` in `/test/node_modules` is there? It should then traverse upward and find it in `../node_modules`.

Reply:
Right, and I think it does! But when I'm spying on a module, I have to tell Jest where it is, which is why I have to say, "look up at the node_modules at the root!"

Reply:
Huh, weird! Sounds good to me then.

Reply:
I know, Jest weirdness strikes again lol